src/model/member.ts

```diff
@@ -1,6 +1,8 @@
 import { Track } from './track';
 
 export type StatusType = 'ATTEND' | 'OFF' | 'IPP' | 'ARMY' | 'COMPLETION' | 'GRADUATE';
+export type StatusLabel = '재학' | '휴학' | '현장실습' | '군 휴학' | '수료' | '졸업';
+export type Status = 'ATTEND' | 'OFF' | 'IPP' | 'ARMY' | 'COMPLETION' | 'GRADUATE' | '재학' | '휴학' | '현장실습' | '군 휴학' | '수료' | '졸업';
```

Comment:
이 타입이 한글&영어가 동시에 쓰일 곳이 있나요?.?
줄인다면 아래처럼 줄일수있어보이긴하네여
```suggestion
export type Status = StatusLabel | StatusType;
```

Reply:
서버에서의 `status`와 렌더링 해야할 `status`가 달라 맵핑을 해줘야 하는데 이때 모든 상태를 나타낼 수 있는 타입이 필요해서 이렇게 사용했습니다. 27491600669e669f5f72683961bf76832590b004